backend/src/tasks/functions.yml

```diff
@@ -33,3 +33,8 @@ updateScanTaskStatus:
           detail:
             clusterArn:
               - ${file(env.yml):${self:provider.stage}-ecs-cluster, ''}
+notifications:
+  handler: src/tasks/notifications.handler
+  events:
+    - schedule: rate(5 minutes)
```

Comment:
I think we should run this every hour instead

Reply:
updated in [ce7aff3](https://github.com/cisagov/crossfeed/pull/1709/commits/ce7aff304818de8b0668fd6632f78ede3d293262)